include/zephyr/bluetooth/audio/pacs.h

```diff
@@ -45,6 +45,18 @@ struct bt_pacs_cap {
 	sys_snode_t _node;
 };
 
+
+/** Structure for registering PACS */
+struct bt_bap_pacs_register_param {
+	bool snk_pac;
+	bool snk_pac_loc;
+	bool src_pac;
+	bool src_pac_loc;
+};
+
+int bt_pacs_register(const struct bt_bap_pacs_register_param *param);
```

Comment:
Since PACS may now be unregistered at boot, we should probably add checks for at least the unicast server to not allow registering ASCS until after PACS, right?
Not sure if we can add anything for the broadcast sink, as that also mandates the use of PACS